.travis.yml

```diff
@@ -65,17 +66,19 @@ rvm:
   - *ruby_2_1
   - *ruby_2_2
   - *ruby_2_3
+  - *ruby_2_4
+  - *ruby_2_5
+  - *ruby_2_6
```

Comment:
it's fine that you want this, but  we're already testing 2.7, so the pr description is out of sync with the intent

Reply:
these lines have already been there for long. here is the last commit prior to our attempt to support ruby 2.7

https://github.com/rails-api/active_model_serializers/blob/6b093c965f75f87d5bbf0510e64b5193f4c6f157/.travis.yml#L65..L67

which got removed from #2368 by mistake

Reply:
I have updated the OP intent. Please let me know if you think we should move some of these changes out of this

Reply:
I actually removed those lines on purpose. Maybe my execution failed, inasmuch as I intend to just test a few of them, in order to reduce the matrix size to what is likely to catch a problem

Reply:
could you please suggest the build matrix that you have in mind? we can try to configure it accordingly